client-base/src/main/kotlin/app/cash/backfila/client/internal/RealBackfilaClientServiceHandler.kt

```diff
@@ -0,0 +1,85 @@
+package app.cash.backfila.client.internal
+
+import app.cash.backfila.client.BackfilaClientLoggingSetupProvider
+import app.cash.backfila.client.UnknownBackfillException
+import app.cash.backfila.client.spi.BackfilaClientServiceHandler
+import app.cash.backfila.protos.clientservice.GetNextBatchRangeRequest
+import app.cash.backfila.protos.clientservice.GetNextBatchRangeResponse
+import app.cash.backfila.protos.clientservice.PrepareBackfillRequest
+import app.cash.backfila.protos.clientservice.PrepareBackfillResponse
+import app.cash.backfila.protos.clientservice.RunBatchRequest
+import app.cash.backfila.protos.clientservice.RunBatchResponse
+import org.apache.commons.lang3.exception.ExceptionUtils
+import wisp.logging.getLogger
+import javax.inject.Inject
+import kotlin.jvm.Throws
+
+/**
+ * The default BackfilaClientServiceHandler. Adds things like basic logging.
+ */
+internal class RealBackfilaClientServiceHandler @Inject constructor(
```

Comment:
did you bind this anywhere? I'm not really seeing a reason to make this an interface with multiple implementations